Pytest Appium Integration (#1)
BrowserStackCE/pytest-appium-browserstack: README.md

```
curl -u "YOUR_USERNAME:YOUR_ACCESS_KEY" \
-X POST "https://api-cloud.browserstack.com/app-automate/upload" \
-F "file=@/path/to/apk/file"
```

Ensure that @ symbol is prepended to the file path in the above request. Please note the `app_url` value returned in the API response. We will use this to set the application under test while configuring the test later on.

**Note**: If you do not have an .apk or .ipa file and are looking to simply try App Automate, you can download and test using our [sample Android app](https://www.browserstack.com/app-automate/sample-apps/android/WikipediaSample.apk) or [sample iOS app](https://www.browserstack.com/app-automate/sample-apps/ios/BStackSampleApp.ipa).

**2. Configure and run your first single test**

Open `single.json` file in `android/run-single-test` folder for Android and `ios/run-single-test` folder for iOS:

- Replace `BROWSERSTACK_USERNAME` & `BROWSERSTACK_ACCESS_KEY` with your BrowserStack access credentials. Get your BrowserStack access credentials from [here](https://www.browserstack.com/accounts/settings)

> - Replace `bs://<app-id>` wkth the URL obtained from app upload step

Reviewer: ```suggestion
- Replace `bs://<app-id>` with the URL obtained from app upload step
```

- Set the deviceName and platformVersion. You can refer our [Capability Generator](https://www.browserstack.com/app-automate/capabilities)

- Run the below command to execute a single Android test on BrowserStack AppAutomate:
    ```
    cd android
    paver run single
    ```

- Run the below command to execute a single iOS test on BrowserStack AppAutomate:
    ```
    cd ios
    paver run single
    ```

**3. Configure and run your parallel test**

- In order to run tests in parallel across different configurations, Open `parallel.json` file in `android/run-parallel-test` folder for Android and `ios/run-parallel-test` folder for iOS

- Replace `BROWSERSTACK_USERNAME` & `BROWSERSTACK_ACCESS_KEY` with your BrowserStack access credentials. Get your BrowserStack access credentials from [here](https://www.browserstack.com/accounts/settings)

- Replace `bs://<app-id>` wkth the URL obtained from app upload step

- Set the deviceName and platformVersion. You can refer our [Capability Generator](https://www.browserstack.com/app-automate/capabilities)
    

> - Run the below command to execute a single Android test on BrowserStack AppAutomate:

Reviewer: ```suggestion
- Run the below command to execute parallel Android test on BrowserStack AppAutomate:
```

```
cd android
paver run parallel
```

> - Run the below command to execute a single iOS test on BrowserStack AppAutomate:

Reviewer: ```suggestion
- Run the below command to execute a parallel iOS test on BrowserStack AppAutomate:
```
